osmaxx/excerptexport/management/commands/purge_old_result_files.py

```diff
@@ -54,10 +54,14 @@ def _remove_old_files(self):
                 export__output_file__file__isnull=False,
             )
             for old_file in old_files:
-                path = old_file.file.path
                 pk = old_file.id
+                success_message = "Output File #{} removed".format(pk)  # default
+                # if a file has never been attached, but still is stale, remove the object.
```

Comment:
based on the times we removed this exact line of code because it seemed unneeded, I made this comment. But now that we have a test for this scenario, removing makes sense :-)